js/layout-javascript/agenda-code.js

```diff
@@ -38,6 +38,7 @@ function DynamicList(id, data, container) {
   this.agendasByDay;
   this.searchedListItems;
   this.dataSourceColumns;
+  this.dateFieldLocation = 'Full Date';
```

Comment:
dateFieldColumnName? `Location` is ambiguous to me

Reply:
@squallstar I imagine it'd be ambiguous (or just downright meaningless) to you because you don't know that there's a concept called `location` in the LFD. It's a predefined variable for **where** a field would be displayed. Search https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/master/js/default-configs/layouts-config.js for the word `location` and see.